javascript/new-src/src/constants.js

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
I see more specifically it's because it only strips /* and not /**. It looks like the Closure Linter is happy with /* too, so if you can change all the JS comments to /* and then fix this line:
https://github.com/OpenLightingProject/ola/blob/master/scripts/enforce_licence.py#L142